frontend/src/screens/wallet/index.tsx

```diff
@@ -43,7 +40,7 @@ function Wallet() {
   return (
     <>
       <AppHeader title="Wallet" description="" />
-      {showMigrateCard && (
+      {showMigrateCard && needsChannels && (
```

Comment:
Why do we still have `showMigrateCard`?

Reply:
I changed that variable to hasHostedBalance. 

1. if hasHostedBalance and needsChannels show copy for transfer funds and open channel (i guess its fine) 
```  <LinkButton to="/channels/first">Transfer Funds</LinkButton>```
2. and then there are other conditionals which shows either "channel reserve unment" low receving copy


or do we want to remove the transfer funds copy here as well. it shows transfer but just goes to first channel flow

Reply:
The idea is to remove the hosted balance because the user does not have a hosted balance any more.